text/0027-co-locate-all-rfcs.md

```diff
@@ -0,0 +1,72 @@
+# Co-locate All Paketo RFCs
+
+## Summary
+
+All individual repository level RFCs that are in the Paketo organization should
+be centralized into the top level RFCS repository.
+
+## Motivation
+
+As it stands currently there are RFCs scattered across various repositories
+throughout the Paketo organization. These RFCs pertain typically to the
+implementation and functionality of either entire language families or even on
+down to implementation buildpack specific behavior. While it is good to have a
+record of technical decisions made for the project and its components, it is
+sometimes hard to search for relative RFCs for a given language family or
+feature. By co-locating all of the RFCs for the project into one repository we
+will collect all of the decision making conversation into one place making it
+easier to access.
+
+## Implementation
+
+Each project team will be given a directory inside of the `rfcs` repository,
+which would be named after the project team. A series of directories with the
```

Comment:
Does "project team" here mean the teams within the Paketo Buildpacks Github org
(https://github.com/orgs/paketo-buildpacks/teams)?

Reply:
Yes so like `Go`, `Java`, `Tooling`, etc.

Reply:
Added a link to the teams in Github.